docs/services/rook-services-guide.md

```diff
@@ -0,0 +1,652 @@
+
+## Deploy Yugastore alongside YugabyteDB
+
+Now that we have a GKE cluster up and running, we can create our YugabyteDB cluster and install Yugastore alongside it.
+
+```bash
+cat > yugastore.yaml <<EOF
+apiVersion: database.crossplane.io/v1alpha1
+kind: PostgreSQLInstance
+metadata:
+  name: yugastore-db
+  namespace: yugastore-app
+  labels:
+    app: yugastore
+spec:
+  writeConnectionSecretToRef:
+    name: yugastore-db-secret
+---
+apiVersion: workload.crossplane.io/v1alpha1
+kind: KubernetesApplication
+metadata:
+  name: yugastore
+  namespace: yugastore-app
+  labels:
+    app: yugastore
+spec:
+  resourceSelector:
+    matchLabels:
+      app: yugastore
+  clusterSelector:
+    matchLabels:
+      app: yugastore
+  resourceTemplates:
+  - metadata:
+      name: yugastore-namespace
+      labels:
+        app: yugastore
+    spec:
+      template:
+        apiVersion: v1
+        kind: Namespace
+        metadata:
+          name: rook-yugastore
+          labels:
+            app: yugastore
+  - metadata:
+      name: yugastore-deployment
+      labels:
+        app: yugastore
+    spec:
+      template:
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          namespace: rook-yugastore
+          name: yugastore
+          labels:
+            app: yugastore
+        spec:
+          selector:
+            matchLabels:
+              app: yugastore
+          template:
+            metadata:
+              labels:
+                app: yugastore
+            spec:
+              containers:
+                - name: yugastore
+                  image: yugabyte/yugastore:latest # Pending merge of https://github.com/yugabyte/yugastore/pull/4
+                  imagePullPolicy: Always
+                  command: ["/usr/local/yugastore/bin/start-for-crossplane.sh"]
+                  env:
+                  - name: DB_HOST
+                    value: "yb-tserver-hello-ybdb-cluster-1.yb-tservers-hello-ybdb-cluster.rook-yugabytedb.svc.cluster.local"
+                  - name: APP_HOST
+                    value: "localhost"
+                  - name: APP_PORT
+                    value: "3001"
+                  ports:
+                    - containerPort: 3001
+                      name: yugastore
+  - metadata:
+      name: yugastore-service
+      labels:
+        app: yugastore
+    spec:
+      template:
+        apiVersion: v1
+        kind: Service
+        metadata:
+          namespace: rook-yugastore
+          name: yugastore
+          labels:
+            app: yugastore
+        spec:
+          ports:
+            - port: 3001
+          selector:
+            app: yugastore
+          type: LoadBalancer
+EOF
+
+kubectl apply -f yugastore.yaml
+```
+
+!! TODO: check successful provisioning (to be updated after CRD defs are finalized) !! 
+!! TODO: make sure loadbalancer IP address is propagated to `KubernetesApplicationResource` `yugastore-service` !!
```

Comment:
This setup exposes a public endpoint for Yugastore. I feel like that makes it a much cooler walk-through (i.e. "you have a public facing production app!"), but it might be worth adding some sort of warning that this will happen / make the `LoadBalancer` service optional.